core/src/retries.rs

```diff
@@ -22,22 +22,91 @@ fn find_io_error(original_error: &reqwest::Error) -> Option<&std::io::Error> {
     None
 }
 
-/// An [`ExponentialBackoff`] with parameters suitable for most HTTP requests. The parameters are
-/// copied from the parameters used in the GCP Go SDK[1].
+#[derive(Copy, Clone, Debug)]
```

Comment:
nit: put derives below doccomment.

Reply:
fixed in 601e8743d57f35a8aaad20cc8bad106226e1fc0f